test/integration/node-exports/src/app.js

```diff
@@ -0,0 +1,5 @@
+function middleware(req) {
+  req.send('Test!');
+}
+
+module.exports = middleware;
```

Comment:
Thanks for adding these fixtures! Can you also add a test that uses them in `test/javascript.js`?